frontend/components/Map.vue

```diff
@@ -7,18 +7,110 @@
       frameborder="0"
       :src="mapUrl.toString()"
     />
+    <div v-if="filterTooltipVisible && appropriateSchool != null" class="filter-popover">
+      <div class="flex flex-col">
+        <span>
+          {{ $t('components.Map.filter_tooltip') }}
+        </span>
+        <span class="font-body-bold">
+          {{ appropriateSchool }}
+        </span>
+      </div>
+      <span class="close" @click="setFilterTooltipVisible(false)" />
```

Comment:
please use appropriate button tag, otherwise it prevents people with reduced mobility or people who are not using mouse to use the website

Reply:
Fixed